### PR TITLE
Add documentation of github repositories

### LINKED ISSUE #3: Github Repos
Covers the number of repos responsible for the EasyEngine v4 project. URL and Purpose purpose of each repo in a tabular format.

### PR DESCRIPTION
**Pull request by kirtangajjar**
Closes #3

### CODE REVIEW

**Comment by rahul286** [comment]
@kirtangajjar Markdown tables would be better. Please use in this as well as other cases also where you can present information in rows & column format.

**Comment by kirtangajjar** [comment]
@rahul286 Done changes as requested

File: handbook/github-repos.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+| [easyengine/config-command](https://github.com/EasyEngine/config-command)   | Used to add/remove config from EasyEngine's global configuration file|
+| [easyengine/admin-tools-command](https://github.com/EasyEngine/admin-tools-command) | Used to enable/disable admin-tools on a site|
+
+## Site Types
+
+There are also seperate repos for "site types". They extend site-command and have logic to create a particular type of site:
+
+| Repository | Descripton |
+| ---------- | ---------- |
+| [easyengine/site-type-php](https://github.com/EasyEngine/site-type-php) | Contains logic to manage PHP site |
```
Comment (rahul286) [inline handbook/github-repos.md]: we also have a repo for html site type

File: handbook/github-repos.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Other Repos
+
+Here are other repos used by EasyEngine team:
+
+| Repository | Descripton |
+| ---------- | ---------- |
+| [easyengine/dockerfiles](https://github.com/EasyEngine/dockerfiles) | Contains all the Dockerfiles used in EasyEngine|
+| [easyengine/easyengine-builds](https://github.com/EasyEngine/easyengine-builds) | Contains build artifacts of EasyEngine. Stable and nightly phars are kept here |
+| [easyengine/installer](https://github.com/EasyEngine/installer) | Contains the installer scripts for EasyEngine |
+| [easyengine/handbook](https://github.com/EasyEngine/handbook) | Contains code to generate automated documentation from docblocks and has other static documentation |
```
Comment (rahul286) [inline handbook/github-repos.md]: This repos name is doc now

**Pull request merged**